docs/book/src/migration/multi-group.md

```diff
@@ -39,36 +38,33 @@ mv apis/v* batch/
 ```
 
 Next, we'll need to update all the references to the old package name. For
-CronJob, that'll be `main.go` and `controllers/cronjob_controller.go`.  If
+CronJob, that'll be `main.go` and `controllers/<group>/cronjob_controller.go`.  If
```

Comment:
If you are finally moving controllers under a GROUP directory, I think you need to add also the equivalent command for them as the bash for the api

> [triple backquote]bash
> mkdir controllers/batch
> mv controller/* controllers/batch/
> [triple backquote]

Didn't write the backquotes because it was formating it as code.

Also, shouldn't the last previous command be `mv apis/v* apis/batch/`?